sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -0,0 +1,184 @@
+-- Basic tests for intervals
+
+select
+  '1' second,
+  2  seconds,
```

Comment:
is this a behavior change? I think we interpret it as `2 as seconds` before, which is an integer literal.

Reply:
yea. Since `days`, `seconds`, ... seems to be non-sql standard and this is out-of-scope in this pr, I'll drop these in this pr.